assets/src/edit-story/components/canvas/test/multiSelectionMovable.js

```diff
@@ -0,0 +1,139 @@
+import Moveable from 'react-moveable';
+import PropTypes from 'prop-types';
+import React, { forwardRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { UnitsProvider } from '../../../../edit-story/units';
+import withOverlay from '../../overlay/withOverlay';
+import CanvasContext from '../context';
+import Selection from '../selection';
+import StoryContext from '../../../app/story/context';
+import TransformProvider from '../../transform/transformProvider';
+
+jest.mock('react-moveable', () => jest.fn(() => ({ children }) => children));
+
+const pageSize = { width: 100, height: 100 };
+
+const element1 = { id: '1', type: 'text' };
+const element2 = { id: '2', type: 'text' };
+
+const WrapperWithRef = ({ children }, ref) => (
+  <div ref={ref}>
+    <TransformProvider>
+      <UnitsProvider pageSize={pageSize}>{children}</UnitsProvider>
+    </TransformProvider>
+  </div>
+);
+const Wrapper = withOverlay(forwardRef(WrapperWithRef));
+
+WrapperWithRef.propTypes = {
+  children: PropTypes.node,
+};
+
+describe('multiSelectionMovable', () => {
+  let updateElementsById;
+  let updateSelectedElements;
+  let target1;
+  let target2;
+  let storyContext;
+  let canvasContext;
+
+  beforeEach(() => {
+    updateElementsById = jest.fn();
+    updateSelectedElements = jest.fn();
+    target1 = document.createElement('div');
+    target2 = document.createElement('div');
+
+    storyContext = {
+      state: { selectedElements: [element1, element2] },
+      actions: { updateElementsById, updateSelectedElements },
+    };
+    canvasContext = {
+      state: {
+        pageSize,
+        nodesById: {
+          '1': target1,
+          '2': target2,
+        },
+      },
+    };
+  });
+
+  function arrange() {
+    return render(
+      <StoryContext.Provider value={storyContext}>
+        <CanvasContext.Provider value={canvasContext}>
+          <Selection />
+        </CanvasContext.Provider>
+      </StoryContext.Provider>,
+      { wrapper: Wrapper }
+    );
+  }
+
+  function performRotatation(rotateTo) {
+    const moveable = Moveable.mock.calls[Moveable.mock.calls.length - 1][0];
+    moveable.onRotateGroupStart({ events: [{ set: () => {} }] });
+    moveable.onRotateGroup({
+      events: [
+        {
+          target: target1,
+          beforeRotate: rotateTo,
+          drag: { beforeTranslate: [0, 0] },
+        },
+        {
+          target: target2,
+          beforeRotate: rotateTo,
+          drag: { beforeTranslate: [0, 0] },
+        },
+      ],
+    });
+    moveable.onRotateGroupEnd({ targets: [target1, target2] });
+  }
+
+  describe('should rotate', () => {
+    const cases = [
+      ['normally', { rotateTo: 45, expectedRotationAngle: 45 }],
+      ['through 360 degrees', { rotateTo: 370, expectedRotationAngle: 10 }],
+      ['through negative', { rotateTo: -370, expectedRotationAngle: 350 }],
+    ];
+
+    it.each(cases)('%p', (_, { rotateTo, expectedRotationAngle }) => {
```

Comment:
...and then here use 'should rotate %s'

So test failures and reporters will print as:

```
Rotation -> should rotate normally
```

And if we ever need more tests related to rotation, they can be added to this `describe` block